torchvision/models/video/resnet.py

```diff
@@ -313,6 +313,9 @@ def _video_resnet(
     "categories": _KINETICS400_CATEGORIES,
     "recipe": "https://github.com/pytorch/vision/tree/main/references/video_classification",
     "_docs": """These weights reproduce closely the accuracy of the paper for 16-frame clip inputs.""",
+    "_frame_rate": 8,
+    "_clips_per_video": 5,
+    "_clip_len": 16,
```

Comment:
The extended tests are failing because those are not part of the schema. You can add them but then we need to review the names. First of all, I agree that all of them should be included here. I'm not sure about the names or structure and I don't have off the top of my head a good recommendation. Let's chat about this tomorrow and solicit feedback from others.

Reply:
As discussed offline, for now we will put this param on the _docs, and we can have a follow up PR for adding this evaluation parameters as meta.